src/tainting/Taint_lval_env.ml

```diff
@@ -480,3 +480,21 @@ let to_string taint_to_str
       pending_propagation_dests "[PENDING PROPAGATION DESTS]"
 
 let seq_of_tainted env = LvalMap.to_seq env.tainted
+
+let find_tainted_lvals_of_common_base { tainted; _ } base =
+  let rec eq_base base1 base2 =
```

Comment:
Better to put this in `IL` or `IL_helpers`, as `equal_base`.

Reply:
done.